counterpartylib/lib/config.py

```diff
@@ -88,7 +88,7 @@
 
 
 DEFAULT_REQUESTS_TIMEOUT = 20  # 20 seconds
-
+DEFAULT_RPC_BATCH_SIZE = 300
```

Comment:
It's a completely arbitrary number. Batch query works well even with 2425 transactions in the block (the max I saw). This parameter is important for people who are using Bitcoin Core with another application (like Counterblock) and should be tweaked with the `rpcthreads` Bitcoin core parameter.